### ISSUE TITLE
Creating new L2s based on an existing L1 chain.

### ISSUE THREAD

**Issue by hamdiallam** [ethereum-optimism/supersim#239]
Right now supersim, creates all the L2s & L1s at once. However, there's a UX journey that would be great to test with supersim, **A new member gets added to the superchain**

We could simulate this with the following flow.
1. Start supersim in the default setting with 1 L2, and 2 L2 chains
2. In a new window, start supersim again with some flags like `--l1.endpoint` and `--l2.starting.port` and `--l2.starting.chainid` which allows new L2s chains to start using the same L1 started in step (1)

This would allow developers to dynamically increase/decrease the number of L2s and set any unique dapp flows. An open question is how the superchain registry works in non fork mode

**Comment by nitaliano** [comment]
Would also be nice to add the ability to do this through admin rpcs as well

**Comment by hamdiallam** [comment]
yep totally agree, maybe the admin rpcs is the right way to do this.

will need to some library to dynamically render terminal output
